test/integration/clusterquota_test.go

```diff
@@ -38,6 +39,10 @@ func TestClusterQuota(t *testing.T) {
 	clusterAdminQuotaClient := quotaclient.NewForConfigOrDie(clusterAdminClientConfig)
 	clusterAdminImageClient := imageclient.NewForConfigOrDie(clusterAdminClientConfig).Image()
 
+	if err := testutil.WaitForClusterResourceQuotaCRDAvailable(clusterAdminClientConfig); err != nil {
```

Comment:
Why not just wait for it in the setup phase, we already lag there, so I don't mind additional few seconds. Especially that you **always** create them so waiting **always** is reasonable.

Reply:
i only want to wait in tests that actually need them